analyzers/src/SonarAnalyzer.Common/Extensions/SyntaxNodeAnalysisContextExtensions.cs

```diff
@@ -39,6 +39,9 @@ internal static class SyntaxNodeAnalysisContextExtensions
         internal static bool IsRedundantPositionalRecordContext(this SyntaxNodeAnalysisContext context) =>
             context.ContainingSymbol.Kind == SymbolKind.Method;
 
+        public static bool IsAzureFunction(this SyntaxNodeAnalysisContext context) =>
+            context.AzureFunctionMethod() is not null;
+
         public static IMethodSymbol AzureFunctionMethod(this SyntaxNodeAnalysisContext context) =>
```

Comment:
NIT: Consider swapping the order of IsAzureFunction and AzureFunctionMethod

Reply:
Both are public and the former one uses the later. It's easier to read (and F# actually enforces this) when both methods are kept in "depends on order".